src/csharp/Microsoft.Spark/ML/Feature/StopWordsRemover.cs

```diff
@@ -0,0 +1,175 @@
+        /// </summary>
+        /// <param name="value">Locale to be used for transform</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        [Since(Versions.V2_4_0)]
+        public StopWordsRemover SetLocale(string value) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setLocale", value));
+
+        /// <summary>
+        /// Gets locale for <see cref="StopWordsRemover"/> transform
+        /// </summary>
+        /// <returns>The locale</returns>
+        [Since(Versions.V2_4_0)]
+        public string GetLocale() => (string)_jvmObject.Invoke("getLocale");
+
+        /// <summary>
+        /// Sets case sensitivity.
+        /// </summary>
+        /// <param name="value">true if case sensitive, false otherwise</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetCaseSensitive(bool value) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setCaseSensitive", value));
+
+        /// <summary>
+        /// Gets case sensitivity.
+        /// </summary>
+        /// <returns>true if case sensitive, false otherwise</returns>
+        public bool GetCaseSensitive() => (bool)_jvmObject.Invoke("getCaseSensitive");
+
+        /// <summary>
+        /// Sets custom stop words.
+        /// </summary>
+        /// <param name="values">Custom stop words</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetStopWords(IEnumerable<string> values) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setStopWords", values));
+
+        /// <summary>
+        /// Gets the custom stop words.
+        /// </summary>
+        /// <returns>Custom stop words</returns>
+        public IEnumerable<string> GetStopWords() =>
+            (IEnumerable<string>)_jvmObject.Invoke("getStopWords");
+
+        /// <summary>
+        /// Check transform validity and derive the output schema from the input schema.
+        /// 
+        /// This checks for validity of interactions between parameters during Transform and
+        /// raises an exception if any parameter value is invalid.
+        ///
+        /// Typical implementation should first conduct verification on schema change and parameter
+        /// validity, including complex parameter interaction checks.
+        /// </summary>
+        /// <param name="value">
+        /// The <see cref="StructType"/> of the <see cref="DataFrame"/> which will be transformed.
+        /// </param>
+        /// <returns>
+        /// The <see cref="StructType"/> of the output schema that would have been derived from the
+        /// input schema, if Transform had been called.
+        /// </returns>
+        public StructType TransformSchema(StructType value) =>
+            new StructType(
+                (JvmObjectReference)_jvmObject.Invoke("transformSchema",
+                    DataType.FromJson(_jvmObject.Jvm, value.Json)));
```

Comment:
nit: per https://github.com/dotnet/spark/blob/master/docs/coding-guidelines/csharp-coding-style.md (Line-break for each method parameters:)
```suggestion
                (JvmObjectReference)_jvmObject.Invoke(
                    "transformSchema",
                    DataType.FromJson(_jvmObject.Jvm, value.Json)));
```